conf/envsubst_template.json

```diff
@@ -0,0 +1,32 @@
+{
+  "port": ${GIT_BRIDGE_PORT:-8000},
+    "rootGitDirectory": "${GIT_BRIDGE_ROOT_DIR:-/tmp/wlgb}",
+    "apiBaseUrl": "${GIT_BRIDGE_API_BASE_URL:-https://localhost/api/v0}",
+    "username": "${GIT_BRIDGE_USERNAME}",
+    "password": "${GIT_BRIDGE_PASSWORD}",
+    "postbackBaseUrl": "${GIT_BRIDGE_POSTBACK_BASE_URL:-https://localhost}",
+    "serviceName": "${GIT_BRIDGE_SERVICE_NAME:-Overleaf}",
+    "oauth2": {
+      "oauth2ClientID": "${GITBRIDGE_OAUTH2_CLIENT_ID}",
```

Comment:
I think we should use either `GITBRIDGE_` or `GIT_BRIDGE_` throughout.

Reply:
Ah, good catch 👍

Reply:
Done.